xml/System.IO.Pipelines/ReadResult.xml

```diff
@@ -112,9 +111,8 @@
         <ReturnType>System.Boolean</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>
-          <see langword="true" /> if the <see cref="T:System.IO.Pipelines.PipeReader" /> is complete; otherwise, <see langword="false" />.</summary>
-        <value>To be added.</value>
+        <summary>Gets a value that indicates whether the current <see cref="M:System.IO.Pipelines.PipeReader.ReadAsync(System.Threading.CancellationToken)" /> operation was completed.</summary>
+        <value><see langword="true" /> if the current <see cref="M:System.IO.Pipelines.PipeReader.ReadAsync(System.Threading.CancellationToken)" /> operation was completed; otherwise, <see langword="false" />.</value>
```

Comment:
Based on the suggestion above:
```suggestion
        <value><see langword="true" /> if the end of the data stream has been reached; otherwise, <see langword="false" />.</value>
```